### PR TITLE
Fix #652 regression

### LINKED ISSUE #684: PSP regression
**Describe the bug**:

#652 introduced a breaking change on some clusters.

**Expected behaviour**:

Successful deployment of the Helm chart.

**Steps to reproduce the bug**:

```bash
helm upgrade --install logging-operator banzaicloud-stable/logging-operator --version 3.9.0
```

**Additional context**:

We, for example, are using k8s 1.18.15 with this AMI: `099720109477/ubuntu/images/hvm-ssd/ubuntu-focal-20.04-amd64-server-20210119.1` Another user reported an issue on 1.20.2 as well (ref: https://github.com/banzaicloud/logging-operator/pull/652#issuecomment-781419351)

**Environment details**:
- Kubernetes version (e.g. v1.15.2): v1.18.15 (and v1.20.2)
- Cloud-provider/provisioner (e.g. AKS, GKE, EKS, PKE etc): kOps on AWS
- logging-operator version (e.g. 2.1.1): chart version 3.9.0
- Install method (e.g. helm or static manifests): helm
- Logs from the misbehaving component (and any other relevant logs):
```
Error: UPGRADE FAILED: cannot patch "psp.logging-operator" with kind PodSecurityPolicy: PodSecurityPolicy.extensions "psp.logging-operator" is invalid: metadata.annotations[seccomp.security.alpha.kubernetes.io/defaultProfileName]: Invalid value: "docker/default,runtime/default": must be a valid seccomp profile
```
- Resource definition (possibly in YAML format) that caused the issue, without sensitive data: n/a

/kind bug

### PR DESCRIPTION
**Pull request by yurrriq**
| Q               | A
| --------------- | ---
| Bug fix?        | yes
| New feature?    | no
| API breaks?     | no
| Deprecations?   | no
| Related tickets | fixes #684, fixes #652
| License         | Apache 2.0

### What's in this PR?

Fix the default seccomp profile name.

### Why?

It was broken in #652.

### Checklist

- [x] Code meets the [Developer Guide](https://github.com/banzaicloud/developer-guide)
- [x] User guide and development docs updated (if needed)
- [x] Related Helm chart(s) updated (if needed)

### CODE REVIEW

File: charts/logging-operator/templates/psp.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ metadata:
   name: psp.logging-operator
   namespace: {{ include "logging-operator.namespace" . }}
   annotations:
-    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'docker/default,runtime/default'
     seccomp.security.alpha.kubernetes.io/allowedProfileNames: 'docker/default,runtime/default'
+    seccomp.security.alpha.kubernetes.io/defaultProfileName:  'runtime/default'
```
Comment (pepov) [inline charts/logging-operator/templates/psp.yaml:9]: Would it make sense to make these two configurable?

**Review by pepov — APPROVED** [review]

**Review by ahma — APPROVED** [review]
Thanks @yurrriq

File: charts/logging-operator/templates/psp.yaml (hunk 1 above)
Comment (yurrriq) [reply charts/logging-operator/templates/psp.yaml:9]: Yes, I think so. But these changes meet our needs for now :)

Comment (yurrriq) [reply charts/logging-operator/templates/psp.yaml:9]: So we don't forget: https://github.com/banzaicloud/logging-operator/issues/688

**Pull request merged**